hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/WritableUpgradeStoreTest.java

```diff
@@ -50,10 +50,10 @@ void commitsUpgradeFileExist() {
         assertFalse(writableUpgradeStates.iterator().hasNext());
 
         writableUpgradeStates.add(file.contents());
-        writableUpgradeFileStates.put(file);
+        writableUpgradeFileStates.put(file.fileId(), file);
 
-        assertTrue(writableUpgradeFileStates.get().fileId().fileNum() == 150L);
-        final var writtenFile = writableUpgradeFileStates.get();
+        assertTrue(writableUpgradeFileStates.get(fileUpgradeFileId).fileId().fileNum() == fileUpgradeFileId.fileNum());
```

Comment:
This is a bit tautological. The RHS would be better as a proper constant.

Reply:
fixed